src/imitation/util/networks.py

```diff
@@ -1,150 +1,33 @@
 """Helper methods to build and run neural networks."""
 
 import collections
-import contextlib
-from typing import Callable, Dict, Iterable, Optional, Sequence, Tuple
+from typing import Callable, Iterable, Optional
 
-import gym
-import tensorflow as tf
-from stable_baselines.common.input import observation_input
-
-LayersDict = Dict[str, tf.layers.Layer]
+from torch import nn
 
 
 def build_mlp(
+    in_size: int,
     hid_sizes: Iterable[int],
     name: Optional[str] = None,
-    activation: Optional[Callable] = tf.nn.relu,
+    activation: Optional[Callable] = nn.ReLU,
     initializer: Optional[Callable] = None,
-) -> LayersDict:
+) -> nn.Module:
     """Constructs an MLP, returning an ordered dict of layers."""
     layers = collections.OrderedDict()
 
     # Hidden layers
+    prev_size = in_size
     for i, size in enumerate(hid_sizes):
-        key = f"{name}_dense{i}"
-        layer = tf.layers.Dense(
-            size, activation=activation, kernel_initializer=initializer, name=key
-        )  # type: tf.layers.Layer
-        layers[key] = layer
+        layers[f"{name}_dense{i}"] = nn.Linear(prev_size, size)  # type: nn.Module
+        prev_size = size
+        if activation:
+            layers[f"{name}_act{i}"] = activation
+            layers.append(activation)
```

Comment:
Oops, the `layers.append()` thing shouldn't be here (this is my fault).

Reply:
Do you want to go ahead and make the fix then? Or tell me exactly which line(s) to delete?

Reply:
It's fixed on my branch. You can just delete `layers.append(activation)` for the same effect.